pkg/generate/app/cmd/newapp_test.go

```diff
@@ -1210,6 +1211,116 @@ func TestRunBuilds(t *testing.T) {
 				return err.Error() == "--dockerfile cannot be used with multiple source repositories"
 			},
 		},
+
+		{
+			name: "successful input image source build with a repository",
+			config: &AppConfig{
+				SourceRepositories: []string{
+					"https://github.com/openshift/ruby-hello-world",
+				},
+				SourceImage:     "centos/mongodb-26-centos7",
+				SourceImagePath: "/src:/dst",
```

Comment:
it would, but this test doesn't actually care about that since it's not actually creating the BuildConfig.  i'll fix it to make it valid.